modules/ts/src/cuda_test.cpp

```diff
@@ -414,6 +414,7 @@ namespace cvtest
 
     void showDiff(InputArray gold_, InputArray actual_, double eps)
     {
+#ifndef WINRT
```

Comment:
I believe this is not needed anymore (runtime exception is not a problem, this function is used for manual debugging).